site/content/contribute/more-info/getting-started/branching.md

```diff
@@ -35,6 +35,35 @@ When your PR is required on a release branch (e.g. for a dot release or to fix a
 If the PR needs to go to other release branches, you can run the command `/cherry-pick release-x.yz` in the PR comments and it will try to cherry-pick it to the branch you specified.
 {{</note>}}
 
+### Manual cherry-pick
+
+If conflicts appear between your pull request (PR) and the cherry-pick target branch, the automated cherry-pick process will fail and will let you know that you need to do a manual cherry-pick. Here are the steps to do so:
+
+1. Pull your original main branch. If your pull request has been merged into master, execute the following command:
+```sh
+git checkout master
+git pull
+```
+2. Repeat the above step for the branch where you want to cherry-pick to:
+```sh
+git checkout release-[VERSION]
+git pull
```

Comment:
Instead of updating your local branches and working from an implicit `HEAD`, I'd recommend fetching origin and working explicitly instead, e.g.:

```
// Update our snapshot of the remote branches
git fetch origin
// Checkout the branch starting at the release branch on origin
git checkout -b manual-cherry-pick-pr-[PR_NUMBER] origin/release-[VERSION]
// find the SHA from, e.g. git log origin/master
git cherry-pick [SHA]
git push -u origin
```

Reply:
Excellent suggestion. Thank you Jesse!